packages/ui/src/components/Table/TableBodyContent/styles.module.css

```diff
@@ -1,6 +1,6 @@
 .row {
     .cell {
-        padding: var(--go-ui-spacing-sm);
+        padding: 12px;
```

Comment:
Let's use design tokens.

Also not sure if "top and bottom lines Indents must be at least 12 px." means adding padding